src/renderer/state.ts

```diff
@@ -171,14 +171,16 @@ export class AppState {
 
           this.isWarningDialogShowing = true;
 
+          const remote = require('electron').remote;
```

Comment:
Could we handle this with IPC directly rather than add an instance of `remote`?

Reply:
Wasn't sure how ya'll felt about `remote` in this repo, happy to switch away. Will update this